### ISSUE TITLE
Popover Invalid Aria Attributes

### ISSUE THREAD

**Issue by OlliTapioNieminen** [radix-ui/primitives#3349]
### Current Behavior

The trigger button of the popover currently includes the attributes aria-expanded and aria-controls.

These attributes are incorrect in the context of a popover acting as a modal.
Popovers (in this implementation) behave as modals because they trap focus and prevent interaction with the rest of the UI until dismissed.

### Expected behavior

Expect Popover to not implement focus trap, and or to not have aria attributes which misleadingly tell assistive technologies that the interaction should be more like an accordion (which will not have focus trap).

### Reproducible example

https://codesandbox.io/p/sandbox/radix-ui-popover-abstracted-example-8jxqn?file=%2Fsrc%2FApp.js

Using key navigation and open the modal by pressing enter on the trigger.  Modal opens however focus is trapped.   

### Suggested solution

 Remove aria-expanded and 	aria-controls from the trigger

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Radix Package(s) |      @radix-ui/react-popover   |   1.0.7    |
| React            | n/a     |     16    |
| Browser          |   Chrome      | 131.0.6778.109         |
| Assistive tech   |     Voice Over/ Sonoma    |  14.7.1   |
| Node             | n/a     |         |
| npm/yarn         |     yarn    |         |
| Operating System |   Sonoma      |  14.7.1       |
